requirements.txt

```diff
@@ -37,7 +37,7 @@ gviz-api==1.10.0
     # via -r requirements.in
 kombu==4.6.11
     # via celery
-mysqlclient==1.4.6
+mysqlclient
```

Comment:
why was this change made?

Reply:
I forgot to undo this one
It will be reverted